### PR TITLE
FIX: add dumb banner to everything in examples/

### PR DESCRIPTION
**Pull request by jklymak**
There is a fancier version of this in the PR list, but this takes care of the worst offenders in `examples/`....

boo

### CODE REVIEW

File: examples/animation/animate_decay.html [hunk 1]
```diff
@@ -40,8 +40,14 @@
     <link rel="prev" title="matplotlib.animation.Animation.to_html5_video" href="../../api/_as_gen/matplotlib.animation.Animation.to_html5_video.html" /> 
 
 
-  </head>
+    </head>
```
Comment (QuLogic) [inline examples/animation/animate_decay.html]: Why this indent?

File: examples/animation/animate_decay.html [hunk 2]
```diff
@@ -40,8 +40,14 @@
...
   <body>
+    <div id="unreleased-message">
+      <link rel="next" title="artist Module" href="../artist_api.html" />
```
Comment (QuLogic) [inline examples/animation/animate_decay.html]: And why the extra `link` (shouldn't it be in `head`?)

Comment (jklymak) [reply examples/animation/animate_decay.html]: Thanks, both should be fixed now - sorry bad cut and paste.

**Comment by jklymak** [comment]
I'll close in lieu of larger website changes

**Pull request closed without merging**